### DISCUSSION TITLE
I am trying to deploy my project to vercel but facing error

### DISCUSSION THREAD

**Discussion by alaminrahman** [prisma/prisma#21558] [Q&A] [answered]
### Question

PrismaClientInitializationError: Prisma has detected that this project was built on Vercel, which caches dependencies. This leads to an outdated Prisma Client because Prisma's auto-generation isn't triggered. To fix this, make sure to run the `prisma generate` command during the build process.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
Mysql
- Node.js version: 
node v18.10.0

- Run `prisma -v` to see your Prisma version and paste it
- "@prisma/client": "^5.4.2",

**Comment by alaminrahman** [comment]
"@prisma/client": "^5.4.2",

**Reply by ludralph** [reply, marked as answer]
Hi @alaminrahman 👋 

Can you please add the `prisma generate script` to the `postinstall` script of your application as recommended in this [documentation](https://www.prisma.io/docs/guides/deployment/serverless/deploy-to-vercel#generate-prisma-during-build)

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by alaminrahman** [reply]
Thanks man. Its working

**Reply by ludralph** [reply]
Awesome. I'm glad i could help 🙌

**Comment by Racine14Creator** [comment]
![image](https://github.com/user-attachments/assets/a5ac2cde-ffbc-44b2-a4c4-da78bc2c0c94)

Guys I have this error on vercel and I don't know how to fix it.
